### PR TITLE
Document `redis-cluster` explicitly in config

### PR DESCRIPTION
**Pull request by pat-s**

### CODE REVIEW

File: docs/content/doc/administration/config-cheat-sheet.en-us.md [hunk 1]
```diff
@@ -492,7 +492,7 @@ Configuration at `[queue]` will set defaults for queues with overrides for indiv
 - `DATADIR`: **queues/**: Base DataDir for storing persistent and level queues. `DATADIR` for individual queues can be set in `queue.name` sections but will default to `DATADIR/`**`common`**. (Previously each queue would default to `DATADIR/`**`name`**.) Relative paths will be made absolute against `%(APP_DATA_PATH)s`.
 - `LENGTH`: **20**: Maximal queue size before channel queues block
 - `BATCH_LENGTH`: **20**: Batch data before passing to the handler
-- `CONN_STR`: **redis://127.0.0.1:6379/0**: Connection string for the redis queue type. Options can be set using query params. Similarly LevelDB options can also be set using: **leveldb://relative/path?option=value** or **leveldb:///absolute/path?option=value**, and will override `DATADIR`
+- `CONN_STR`: **redis://127.0.0.1:6379/0** or **redis+cluster://127.0.0.1:6379/0**: Connection string for the redis queue type. Options can be set using query params. Similarly LevelDB options can also be set using: **leveldb://relative/path?option=value** or **leveldb:///absolute/path?option=value**, and will override `DATADIR`
```
Comment (delvh) [inline docs/content/doc/administration/config-cheat-sheet.en-us.md]: The text in bold is the default value.
The default value doesn't change.

Comment (delvh) [reply docs/content/doc/administration/config-cheat-sheet.en-us.md]: The changes should be mirrored to `app.example.ini` by the way.

File: custom/conf/app.example.ini [hunk 2]
```diff
@@ -1697,8 +1697,9 @@ ROUTER = console
 ;; For "memory" only, GC interval in seconds, default is 60
 ;INTERVAL = 60
 ;;
-;; For "redis" and "memcache", connection host address
+;; For "redis", "redis-cluster" and "memcache", connection host address
 ;; redis: `redis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`
+;; redis-cluster: ``redis+cluster://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`
```
Comment (delvh) [inline custom/conf/app.example.ini]:
```suggestion
;; redis-cluster: `redis+cluster://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`
```

File: docs/content/doc/administration/config-cheat-sheet.en-us.md [hunk 3]
```diff
@@ -484,7 +484,7 @@ Configuration at `[queue]` will set defaults for queues with overrides for indiv
 - `DATADIR`: **queues/common**: Base DataDir for storing level queues. `DATADIR` for individual queues can be set in `queue.name` sections. Relative paths will be made absolute against `%(APP_DATA_PATH)s`.
 - `LENGTH`: **100**: Maximal queue size before channel queues block
 - `BATCH_LENGTH`: **20**: Batch data before passing to the handler
-- `CONN_STR`: **redis://127.0.0.1:6379/0**: Connection string for the redis queue type. Options can be set using query params. Similarly, LevelDB options can also be set using: **leveldb://relative/path?option=value** or **leveldb:///absolute/path?option=value**, and will override `DATADIR`
+- `CONN_STR`: **redis://127.0.0.1:6379/0**: Connection string for the redis queue type. For `redis-cluster` use `redis+cluster://127.0.0.1:6379/0`. Options can be set using query params. Similarly LevelDB options can also be set using: **leveldb://relative/path?option=value** or **leveldb:///absolute/path?option=value**, and will override `DATADIR`
```
Comment (delvh) [inline docs/content/doc/administration/config-cheat-sheet.en-us.md]:
```suggestion
- `CONN_STR`: **redis://127.0.0.1:6379/0**: Connection string for the redis queue type. For `redis-cluster` use `redis+cluster://127.0.0.1:6379/0`. Options can be set using query params. Similarly, LevelDB options can also be set using: **leveldb://relative/path?option=value** or **leveldb:///absolute/path?option=value**, and will override `DATADIR`
```

**Review by delvh — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**